testing/munin/APKBUILD

```diff
@@ -0,0 +1,72 @@
+# Contributor: Stefan Wagner <stw@bit-strickerei.de>
+# Maintainer: Stefan Wagner <stw@bit-strickerei.de>
+pkgname=munin
+pkgver=2.0.25
+pkgrel=0
+pkgdesc="A distributed monitoring/graphing tool"
+url="http://munin-monitoring.org/"
+arch="noarch"
+license="GPL"
+perl_modules="perl-rrd perl-net-snmp perl-log-log4perl perl-html-template
+perl-net-ssleay perl-net-server perl-date-manip perl-io-socket-inet6
+perl-file-copy-recursive perl-fcgi perl-uri"
+depends="$pkgname-node bash rrdtool perl $perl_modules"
+depends_dev="perl perl-module-build"
+makedepends="$depends_dev"
+install="$pkgname.pre-install $pkgname.post-install $pkgname.post-upgrade"
+subpackages="$pkgname-node"
+source="http://downloads.munin-monitoring.org/munin/stable/$pkgver/$pkgname-$pkgver.tar.gz
+Makefile.config munin.cron.d munin-node.initd munin.logrotate munin-node.logrotate"
+
+_builddir="$srcdir/$pkgname-$pkgver"
+build() {
+  cd "$_builddir"
+  cp ../Makefile.config .
+  make -j1 || return 1
+}
+
+package() {
+  cd "$_builddir"
+  make DESTDIR="$pkgdir" install-master-prime || return 1
+  rm -rf $pkgdir/usr/share/man
+
+  install -m644 -D "$srcdir"/munin.cron.d \
+    "$pkgdir"/etc/munin/munin.cron.sample || return 1
+  install -m644 -D "$srcdir"/munin.logrotate \
+    "$pkgdir"/etc/logrotate.d/munin || return 1
+}
+
+node() {
+  depends="bash rrdtool perl $perl_modules"
+  install="$subpkgname.pre-install $subpkgname.post-install $subpkgname.post-upgrade"
+
+  mkdir -p "$subpkgdir"
+
+  cd "$_builddir"
+  make DESTDIR="$subpkgdir" install-common-prime install-node-prime install-plugins-prime || return 1
+  rm -rf $subpkgdir/usr/share/man || return 1
```

Comment:
`$subpkgdir` should be always quoted, especially when it’s used as an argument for `rm -rf`…